src/data/nav.ts

```diff
@@ -1,16 +1,21 @@
-import publisherIcon from '@assets/publishers-icon.png?inline'
-import supporterIcon from '@assets/supporters-icon.png?inline'
-import devIcon from '@assets/developer-icon.png?inline'
+import developerIcon from '@assets/dropdown-developer.png?inline'
+import publisherIconInactive from '@assets/dropdown-publisher-inactive.png?inline'
+import publisherIconActive from '@assets/dropdown-publisher-active.png?inline'
+import supporterIconInactive from '@assets/dropdown-supporter-inactive.png?inline'
+import supporterIconActive from '@assets/dropdown-supporter-active.png?inline'
 
 export const TOOL_LABEL_KEYS = ['nav.pub-tools', 'nav.supp-tools', 'nav.dev-tools'] as const;
 export type ToolLabelKey = typeof TOOL_LABEL_KEYS[number];
 
 interface ToolsSubmenuItem {
-    image: {
+    imageInactive: {
         src: string,
         width: number,
         height: number
     },
+    imageActive: {
+        src: string
+    },
```

Comment:
Consider using: `image` and `imageInteractive` (or something).
Active/Inactive can be confusing.